pkg/driver/node_linux.go

```diff
@@ -0,0 +1,109 @@
+// +build linux
+
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package driver
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"golang.org/x/sys/unix"
+	"k8s.io/klog"
+)
+
+// findDevicePath finds path of device and verifies its existence
+// if the device is not nvme, return the path directly
+// if the device is nvme, finds and returns the nvme device path eg. /dev/nvme1n1
+func (d *nodeService) findDevicePath(devicePath, volumeID string) (string, error) {
+	exists, err := d.mounter.PathExists(devicePath)
+	if err != nil {
+		return "", err
+	}
+
+	// If the path exists, assume it is not nvme device
```

Comment:
Can we put a link to the source of this assumption and/or explain why it's safe to do so?

Reply:
i have no clue, keep in mind this is copy/pasted from node.go

Reply:
I think this is it, will link: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html#available-ec2-device-names

Reply:
Thanks. You can do it on the next PR if you like, up to you.